language/move-lang/tests/move_check/expansion/global_access_pack.exp

```diff
@@ -2,10 +2,8 @@ error:
 
    ┌── tests/move_check/expansion/global_access_pack.move:3:13 ───
    │
- 3 │         ::S { }
-   │             ^ Unexpected '{'
-   ·
  3 │         ::S { }
    │             - Expected '('
+   │             ^ Unexpected '{'
```

Comment:
Similar issue to the other error messages. These got *a lot* worse. I don't think I'd be happy with this upgrade, and we will have to stay at the old version/fork